signup/signupController.js

```diff
@@ -106,19 +110,41 @@ define('signupController', [], function() {
 				"siteId": $scope.sharedProperties.selectedSite
 			};
 
-			SignupService.storeAppointments(data).then((response) => {
-				if (typeof response !== 'undefined' && response && response.success){
-					document.body.scrollTop = document.documentElement.scrollTop = 0;
-					$scope.appointmentId = response.appointmentId;
-					$scope.successMessage = $sce.trustAsHtml(response.message);
+			const firstName = $scope.data.firstName || '';
+			const lastName = $scope.data.lastName || '';
+			SignupService.getExistingClientAppointments(firstName, lastName).then((response) => {
+				if (typeof response !== 'undefined' && response && response.success) {
+					// response.numberExistingAppointments === false when query is empty because none exist
+					if(response.numberExistingAppointments === false || response.numberExistingAppointments['numberExistingAppointments'] === '0') {
```

Comment:
Should maybe do this `=== false` on the BE and convert it to a zero. Also, does this really return a stringified number? It doesn't just return `0`?